DataManager/src/Tests/VersionTester.java

```diff
@@ -0,0 +1,73 @@
+package Tests;
+
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.awt.Rectangle;
+
+import javax.swing.JFrame;
+import javax.swing.JTextPane;
+
```

Comment:
These tests were just built for my personal tests and to assist testers, they have been removed from the current build